build.gradle

```diff
@@ -121,17 +102,15 @@ def radixVersion() {
     // Ensure version is safe for gradle's purposes
     // (we do further sanitizing for debian's purposes in debSafeVersionName())
     version = version.replaceAll('/', '~')
+
+    print "radixdlt-version:$version"
```

Comment:
Out of interest, what is this print statement for?

Reply:
That's used in [this](https://github.com/radixdlt/babylon-node/pull/877/files/632afb4484e0e810162e5e2aceb7ceeaba410b58#diff-728762f38670ff7bf72d3b2e830bd93839f109400cb2506413596d2aa3d04b93) action. But you suggested we might not need the action at all. So I'll look into it to see what it's used for.

Reply:
That was for debug - it's gone now.